internal/os/volume/api.go

```diff
@@ -19,9 +19,9 @@ func New() VolAPIImplementor {
 }
 
 func runExec(cmd string) ([]byte, error) {
-	klog.V(5).Infof("Running command: %s", cmd)
+	klog.V(4).Infof("Running command: %s", cmd)
```

Comment:
Would we end up logging any password when logging the command as a whole ? Guessing that only env has the sensitive info like that we wouldn't print it here. Wanted to confirm ?

Reply:
I removed the log. Is that ok?